Spigot/src/main/java/net/notfab/hubbasics/spigot/nms/NMSVersion.java

```diff
@@ -40,6 +40,8 @@ public CraftBukkitVersion getRunningVersion() {
     }
 
     public String getRunningNMS() {
+        if (this.getRunningVersion().getName().equals("Unsupported"))
```

Comment:
I'd prefer if this check was made using enums and a bit of variable reuse.

This is my proposed approach:
```java
    public String getRunningNMS() {
        CraftBukkitVersion version = this.getRunningVersion();
        if (version == CraftBukkitVersion.UNSUPPORTED) {
            String packageName = Bukkit.getServer().getClass().getPackage().getName();
            return packageName.substring(packageName.lastIndexOf(".") + 1);
        } else {
            return version.getNmsNames()[0];
        }
    }
```

Reply:
I will incorporate this into the pull request. My original thought was very similar to yours, but I saw it was gonna take some tweaking to make it fit into the plugin.